logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/creating/filesystem/hints/hints.kt

```diff
@@ -45,6 +48,18 @@ fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withFileAttributesFailu
         }
     }
 
+fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withPathAttributesFailureHint(
+    provider: SubjectProvider<Path>
+): Descriptive.DescriptionOption<DescriptiveAssertionWithFailureHint.FinalStep> =
+    withFailureHintBasedOnDefinedSubject(provider) { path ->
+        explainForResolvedLink(path) {
```

Comment:
I don’t think `explainForResolvedLink` is correct here. The assertion does not resolve symlinks (right?), so there should also be no explanation about the resolved target. The assertion should only be about the path itself, regardless of the file system content.

Reply:
Simplified to
```kotlin
fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withPathAttributesFailureHint(
    provider: SubjectProvider<Path>
): Descriptive.DescriptionOption<DescriptiveAssertionWithFailureHint.FinalStep> =
    withFailureHintBasedOnDefinedSubject(provider) { path ->
        listOf(describeWas(path.type))
            .let {
                assertionBuilder.explanatoryGroup.withDefaultType
                    .withAssertions(it)
                    .build()
            }
    }

```